### PR TITLE
Using a google calendar api for getting future rooms list.

### PR DESCRIPTION
**Pull request by AlexOrd**

### CODE REVIEW

File: src/actions/calendarDataActions.js [hunk 1]
```diff
@@ -18,7 +34,15 @@ export function loadEvents() {
...
 }
-export const setCalendarData = calendar => ({
-  type: types.SET_CALENDAR,
-  calendar
-});
+
+export function getCalendarList() {
+  return function(dispatch) {
+    eventsAPI.calendarList().then(response => {
+      const list = response.result.items;
```
Comment (MariiaPolova) [inline src/actions/calendarDataActions.js:41]: you can avoid using temporary vars

File: src/components/ChooseRoom/ChooseRoom.js [hunk 2]
```diff
@@ -63,52 +63,38 @@ const styles = {
...
 };
 
-const ChooseRoom = (propsList) => {
-  const { props, calendars, classes, dispatch } = propsList;
+class ChooseRoom extends React.Component {
+  constructor(props) {
+    super(props);
 
-  const handleClick =(roomData, index) => {
+    this.selectRoomCalendar = this.selectRoomCalendar.bind(this);
```
Comment (MariiaPolova) [inline src/components/ChooseRoom/ChooseRoom.js]: no need to bind

**Review by MariiaPolova — APPROVED** [review]
seems cool! would like to see it in action ;)

File: src/actions/calendarDataActions.js (hunk 1 above)
Comment (AlexOrd) [reply src/actions/calendarDataActions.js:41]: Ok )

**Pull request merged**